README.md

```diff
@@ -1 +1,76 @@
-# angular-component-method-readme
\ No newline at end of file
+# Bonus: Using the component() method
+
+## Overview
+
+Angular 1.5 brought the awesome `.component` directive. Don't worry, the learning curve isn't massive - it's basically a simplified directive!
+
+## Objectives
+
+- Describe component() and directive() differences
+
+## component()
+
```

Comment:
Here, it would be interesting to have a description about web components in general and how they're used in other frameworks (React, Ember, etc. )